webui/doc/README-INSTALLATION.md

```diff
@@ -1,3 +1,5 @@
+
 INSTALLATION
 ============
+
 Please see: https://docs.bareos.org/IntroductionAndTutorial/InstallingBareosWebui.html
```

Comment:
Maybe better to use the :ref:`section-webui` so it will point to the right page (actually BareosWebui.html)

Reply:
that won't work. It's markdown not ReST.